linera-client/src/client_options.rs

```diff
@@ -773,6 +774,10 @@ pub enum ClientCommand {
         /// Path to the Wasm file for the application "service" bytecode.
         service: PathBuf,
 
+        /// The virtual machine runtime to use.
+        #[arg(long)]
+        vm_runtime: Option<VmRuntime>,
```

Comment:
nit: We could also handle the default here.

Reply:
You are correct, thank you.